app/screens/home/channel_list/index.tsx

```diff
@@ -79,18 +74,12 @@ const ChannelListScreen = (props: ChannelProps) => {
                 style={styles.content}
                 edges={['bottom', 'left', 'right']}
             >
+                <ServerIcon/>
                 <Animated.View
                     style={[styles.content, animated]}
                 >
-                    <TeamSidebar/>
-                    <View style={[styles.flex, {alignItems: 'center', justifyContent: 'center'}, tabletSidebarStyle]}>
-                        <Text
-                            onPress={() => goToScreen('Channel', '', undefined, {topBar: {visible: false}})}
-                            style={{fontSize: 20, color: theme.centerChannelColor}}
-                        >
-                            {'Channel List'}
-                        </Text>
-                    </View>
+                    <TeamSidebar iconPad={true}/>
+                    <ChannelList iconPad={false}/>
```

Comment:
Add a comment to handle the `iconPad` prop in a later PR with the multi-server stuff.